data/formations_basic.txt

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2019-2021 by Peter van der Meer
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+# Formation where ships fly in a diagonal line left behind the formation leader.
+formation "Echelon (left) medium"
+	position -150 150
+		repeat -150 150
+
+
+# Formation where ships fly in a diagonal line right behind the formation leader.
+formation "Echelon (right) medium"
+	position 150 150
+		repeat 150 150
+
+
+# Formation where ships form a line right and left of the formation leader.
+formation "Line (sides) medium"
+	flippable x y
+	rotatable 180
+	position -200 0
+		repeat -200 0
+	position 200 0
+		repeat 200 0
+
+
+# Formation where ships form a line behind the formation leader.
+formation "Line (trailing) medium"
+	flippable y
+	position 0 200
+		repeat 0 200
+
+
+# Formation with two legs extending on a 30 degree angle behind the lead ship.
+# For interceptor-sized escorts
+formation "Vic large"
+	flippable y
+	position -200 115
+		repeat -200 115
+	position 200 115
+		repeat 200 115
+
+
+# Formation with two legs extending on a 30 degree angle behind the lead ship.
+# For warship-sized escorts
+formation "Vic medium"
```

Comment:
```suggestion
formation "Vic"
```

Reply:
Actually, I think this could stay since we have three different versions. Maybe it should be `"Vic (small/medium/large)"`, similar to the `"Line (trailing)"`?

Reply:
Well, this is quite a fundamental choice.

Do we go for automatic scaling?
Then we should have only one version for each formation, and then the `small` and `large` versions of Vic need to be removed.

The simplest version of automatic scaling that I can think of is a `scaling` keyword that looks like:
```
formation "Vic"
	scaling
		"participants size" / 125
	flippable y
	position -150 87
		repeat -150 87
	position 150 87
		repeat 150 87
```

If we don't go for automatic scaling, then we should create small/medium and large versions for many of the formations. I don't see "Vic" as special here, because the others would also get extra versions.

My suggestion for now is to remove the `medium` keyword and delete the small and large versions. Then we can still choose in the future if we want automatic scaling or not.

Reply:
Yes, that seems to be the best option for now.

Reply:
I removed the small and large version for now, because I expect that either:
- we will get automatic scaling of formations in the future (this seems likely)
- or we will need to create small and large versions, but I'm sure we will still be able to create alternative versions with proper names even if we don't add the postfixes now.